js/scenes/FoodPlanningScreen/FoodPlanningScreenContainer.js

```diff
@@ -28,30 +23,41 @@ class FoodPlanningScreenContainer extends Component {
       ),
     }),
   };
+  constructor() {
+    super();
+    this.state = {
+      potluckFood: {}
+    };
+  }
 
-  render() {
+  addPotluckItem = (potluckItem) => {
+    const { potluckFood } = this.state;
+    const exists = Object.prototype.hasOwnProperty.call(potluckFood, potluckItem);
 
-    console.log('courses is...', this.props.courses)
+    this.setState({ potluckFood: { ...potluckFood, [potluckItem]: exists ? potluckFood[potluckItem] + 1 : 1 } });
```

Comment:
So we're maintaining this in component state, not redux? When does this state get dispatched? It'd be preferable to just use Redux. (unless you have a good reason we shouldn't)